test/cli/checkers-testing/test1.cpp

```diff
@@ -0,0 +1,4 @@
+int main(int argc)
+{
```

Comment:
this source file is not required. you can reuse one of the existing sourcefiles.